package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "eslint --ignore-path .gitignore --ext ts,tsx,js .",
     "prepare": "husky install",
     "test:unit": "vitest run --config vite.config.unit.mjs",
-    "test:functional": "run-p -r preview test:functional:vitest",
+    "test:functional": "echo 'No functional tests' || run-p -r preview test:functional:vitest",
```

Comment:
The focus-visible test that was removed was the only functional test, meaning the command as it was would fail with that test removed https://github.com/cloudscape-design/chat-components/pull/15/files#diff-4e7974af0586a2d82ba16004a195514b47103208e784fb3950c976954331c825

Reply:
can we remove this script if it is not used then?

Reply:
I was thinking to keep it so that it can easily be added back if we want functional tests for future components. But I guess git history also works for that if you feel strongly about removing it